src/DataValues/DecimalValue.php

```diff
@@ -58,7 +58,9 @@ public function __construct( $value ) {
 		}
 
 		$value = trim( $value );
-
+		if ( $value !== '' && $value[0] !== '-' && $value[0] !== '+' ) {
```

Comment:
I added a few more test cases in #70 but could not find a way to outsmart the current implementation, so it is fine for now. It could, in theory, become a problem when we want to have support for [U+2212](https://en.wikipedia.org/wiki/Plus_and_minus_signs#Minus_sign).